src/app/app.component.ts

```diff
@@ -22,12 +24,14 @@ import { HeaderLandingComponent } from './landing-page/header-landing/header-lan
     HeaderComponent,
     ReactiveFormsModule,
     FooterComponent,
-    HeaderLandingComponent
+    HeaderLandingComponent,
+    AccSuccessComponent
```

Comment:
Is "Acc" stands for "Accept", "Accumulate", or "Account"? Please follow the best naming practices and don't use shortcuts like that